code/controllers/subsystem/shuttle.dm

```diff
@@ -273,10 +273,10 @@ SUBSYSTEM_DEF(shuttle)
 		CRASH("Emergency shuttle block was called, but missing a value for the lockout duration")
 	if(admin_emergency_no_recall)
 		priority_announce(
-			text = "Emergency shuttle uplink interference detected, shuttle call disabled while the system reinitializes. Estimated restore in [DisplayTimeText(lockout_timer, round_seconds_to = 60)].",
-			title = "Uplink Interference",
+			text = "Acil durum mekik uplink paraziti tespit edildi, sistem yeniden başlatılırken mekik çağrısı devre dışı bırakıldı. Tahmini geri yüklenme süresi: [DisplayTimeText(lockout_timer, round_seconds_to = 60)].",
```

Comment:
```suggestion
			text = "Acil durum mekiği uplink paraziti tespit edildi, sistem yeniden başlatılırken mekik çağrısı devre dışı bırakıldı. Tahmini geri yüklenme süresi: [DisplayTimeText(lockout_timer, round_seconds_to = 60)].",
```

Reply:
uplink için bir şey bulmalı mıyız? yoksa uplink olarak kalmalı mı?